File: app/src/main/java/com/bear2b/sampleapp/ui/view/fragments/HistoryFragment.java
package com.bear2b.sampleapp.ui.view.fragments;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.bear2b.sampleapp.R;
import com.bear2b.sampleapp.ui.view.activities.AdvancedSampleActivity;

public class HistoryFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_history, container, false);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        view.findViewById(R.id.sample_history_item).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ((AdvancedSampleActivity) getActivity()).showArSceneWithoutTracking(226620);
            }
        });
    }

    @Override
    public void onResume() {
        super.onResume();
        ((AdvancedSampleActivity) getActivity()).getHandler().cleanView();
    }

    public static HistoryFragment newInstance() {
        return new HistoryFragment();
    }
}

File: app/src/kotlin/java/com/bear2b/sampleapp/ui/view/fragments/HistoryFragment.kt
package com.bear2b.sampleapp.ui.view.fragments

import android.os.Bundle
import android.support.annotation.Nullable
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.bear2b.sampleapp.R
import com.bear2b.sampleapp.ui.view.activities.AdvancedSampleActivity
import kotlinx.android.synthetic.main.fragment_history.*

class HistoryFragment : Fragment() {

    @Nullable
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.fragment_history, container,false)

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        sample_history_item.setOnClickListener { (activity as AdvancedSampleActivity).showArSceneWithoutTracking(226620) }
    }

    override fun onResume() {
        super.onResume()
        (activity as AdvancedSampleActivity).bearHandler.cleanView()
    }

    companion object {
        fun newInstance() = HistoryFragment()
    }

}